src/dabplus-on-air-processing.liq

```diff
@@ -39,10 +39,20 @@ output =
     input
   end
 
-# send the processed signal to the encoder
+# send the processed signal to the ODR encoder
+odr_encoder =  string.concat(
+  separator=" ", 
+  [
+    encoder_bin, 
+    "--input=stdin",
+    "--output=#{encoder_output}",
```

Comment:
You can add more via `encoder_args` for now. I'll refactor it to support an array of items when it gets unwieldy.